pyproject.toml

```diff
@@ -1,3 +1,66 @@
+[tool.poetry]
+name = "starlette-context"
+version = "v0.3.6a0"
+description = "Middleware for Starlette that allows you to store and access the context data of a request. Can be used with logging so logs automatically use request headers such as x-request-id or x-correlation-id."
+authors = ["Tom Wojcik <starlette-context-pkg@tomwojcik.com>"]
```

Comment:
@hhamana lmk if you want to be added as an author and if so, what do you want me to put in here? You're a maintainer on GH anyway.